backend/models/office_hours/event_type.py

```diff
@@ -12,16 +12,16 @@ class OfficeHoursEventType(Enum):
     Determines the type of an office hours event.
     """
 
-    OFFICE_HOURS = 0
-    TUTORING = 1
-    REVIEW_SESSION = 2
+    OFFICE_HOURS = "Office Hours"
+    TUTORING = "Tutoring"
+    REVIEW_SESSION = "Review Session"
```

Comment:
I am slightly concerned this is going to cause a migration and one that will take some care to pull off (I would actually need to think through exactly how...) since we already have a column storing this data in an int form (I believe). The process of changing to a string/text and preserving the data from SS1/SS2 may be tricky. For now, maybe we should leave this as ints and make this something to come back to. Ultimately, I could see this being converted to an entity that courses are able to control and customize.

Reply:
I think this concern still holds here for these enums.